packages/vue-i18n/src/components/Translation.ts

```diff
@@ -27,6 +27,11 @@ export interface TranslationProps extends BaseFormatProps {
    * The Plural Choosing the message number prop
    */
   plural?: number | string
+  /**
+   * @remarks
+   * An existing i18n Composer instance to use for translating
+   */
+  i18n?: Composer
```

Comment:
I think this issue is not only with the `Translation` component but also with the `DatetimeFormat` and `NumberFormat` components.
 
I hope that you could define it to `BaseFormatProps` interface and `baseFormatProps` prop. 
https://github.com/intlify/vue-i18n-next/blob/master/packages/vue-i18n/src/components/base.ts

Reply:
Done! 😃